examples/gallery/embellishments/legend.py

```diff
@@ -4,7 +4,12 @@
 
 The :meth:`pygmt.Figure.legend` method can automatically create a legend for
 symbols plotted using :meth:`pygmt.Figure.plot`. Legend entries are only
-created when the ``label`` parameter is used.
+created when the ``label`` parameter is used. For more complicated legends,
+including legends with multiple columns, users have to write an ASCII file
+with instructions for the layout of the items in the legend and pass it
```

Comment:
```suggestion
with instructions for the layout of the legend items and pass it
```

Reply:
I simply used the upstream GMT documentation at https://docs.generic-mapping-tools.org/dev/legend.html#optional-arguments, but I am fine with this change 🙂.